locallib.php

```diff
@@ -50,6 +52,7 @@
 // Max number of sessions available in the warnings set form to trigger warnings.
 define('ATTENDANCE_MAXWARNAFTER', 100);
 
+
```

Comment:
more extra carriage returns - can you please check you're not adding extra ones in places you have not changed any other code?